app/views/appointments/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<h3>Your appointment</h3>
```

Comment:
You say "your appointments" but you display ALL the appointments, from every user! you need to restrict to only the appointments from the current user